### ISSUE TITLE
Global window "click" events get removed when removing element from group

### ISSUE THREAD

**Issue by epaminond** [heroku/ember-sortable#144]
Hi, thanks for useful package! :)
I've got a weird issue with it, but not sure where it's coming from.

Say I have a component with property:
```javascript
export default Ember.Component.extend({
  selectedDataFields: Ember.A([
    { group: 'Hrs', name: 'Actual Hrs Approved' },
  ],
  //...
  actions: {
    removeDataField(field) {
      // const clickOutsideHandler = $._data(window, 'events').click[0].handler;
      this.get('selectedDataFields').removeObject(field);
      // Ember.run.next(() => $(window).on('click', clickOutsideHandler));
    },
    //...
  },
});
```

And here's part of it's template:
```
table.border-table.mod-full-width.mod-sortable-list
  = sortable-group tagName="tbody" onChange="reorder" as |group|
    each selectedDataFields key="@index" as |field index|
      = sortable-item tagName="tr" model=field group=group handle=".handle"
        td
          %span.handle
            %span.drag-me
          .d-ib.f-z-12 = field.name
          .delete-sortable-item.t-c-light{action 'removeDataField' field} ×
```

When `removeDataField` action gets fired all window 'click'-events (checking via `$._data(window, 'events').click`) get removed. I've made a workaround for that (see commented out code in action) but as of my tests removing `sortable-group` from the template fixes this either.

Tested on 1.9.3.

**Comment by nerdyworm** [comment]
I'm running into something similar.  I believe that https://github.com/jgwhite/ember-sortable/blob/master/addon/mixins/sortable-item.js#L297-L298 is the issue.  `click` needs to be namespaced to `click.ember-sortable` or something like that.

**Comment by jgwhite** [comment]
@nerdyworm sounds like a pragmatic fix. Fancy submitting a PR?

**Comment by nerdyworm** [comment]
There is already a PR for this: https://github.com/jgwhite/ember-sortable/pull/149

The fork is working quite well for us.  I'd like to see it merged if possible :)

**Comment by jgwhite** [comment]
Looks like this should have been automatically closed by #149
